### ISSUE TITLE
pagination issue with reviews & comments

### ISSUE THREAD

**Issue by andrewjs18** [DamnFineShave/mybb-CommunityReviews#26]
this might possibly be classified as a bug, though I'm not sure.

if the display order is set to 'newest to oldest' (I assume this is for everything) and the review goes to a 2nd page and you have you have a lot more recent comments than reviews, it'll start mixing the two together.  see this review as an example: https://dfs.andrewshemo.com/reviews-product-6-aaaaaaaaaa

the reviews should always show up before the comments, regardless of the pagination, I'm thinking...although I'm not sure how you'd go about handling what to show when something gets to page 2.  would you duplicate the reviews on every page so they're on top?

**Comment by andrewjs18** [comment]
@Devilshakerz @leefish 

do you have any recommendations on 'fixing' this issue?

I'm trying to find a working example of something that might suffice.

**Comment by andrewjs18** [comment]
just to note...we decided to go with a format similar to that used on dpreview.com, as seen here: https://www.dpreview.com/reviews/sigma-50mm-f1-4-dg-hsm

the different pages listed in the table of contents in the link above would be the pagination for the reviews (I think we settled on 4 per page) and the comments would load below them and have their own separate pagination.  

comments on any page after page 1 can be loaded via ajax, I believe, without requiring an entire page refresh; changing review pages will probably require a page refresh.  if we go to page 2 in the reviews while we're on page 5 of comments, the comments will get reset back to page 1.  this is how it also works on dpreview.com

@Devilshakerz - agreed?
